identity-wallet/src/state/credentials/mod.rs

```diff
@@ -81,7 +94,10 @@ impl From<CredentialFormats<WithCredential>> for VerifiableCredentialRecord {
                     )
                 };
 
-                let issuance_date = credential_display["issuanceDate"].clone();
+                let mut issuance_date = String::new();
+                if !credential_display["issuanceDate"].is_null() {
+                    issuance_date = credential_display["issuanceDate"].clone().to_string().replace('\"', "");
+                }
```

Comment:
This looks a bit over-complicated. The same functionality can be achieved using:
```rust
    let issuance_date = credential_display["issuanceDate"].as_str().unwrap_or_default();
```